config/common-variables.tf

```diff
@@ -132,3 +132,12 @@ variable "enable_inspector" {
   type        = bool
   default     = false
 }
+
+locals {
+  regions = ["us-east-1", "us-east-2", "global"]
+
+  filtered_region = [for region in local.regions : region if can(regex(region, "${path.cwd}"))][0]
```

Comment:
Would you help me understand why we need to do this? BTW I think it would be a good idea to leave that in a comment line here as well.